source/2019-09-27-the-ember-times-issue-117.md

```diff
@@ -21,13 +22,13 @@ READMORE
 
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
+## [A newly adopted Ember Addon 🐣](https://twitter.com/Alonski/status/1177577454711582720)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
+[Adopted Ember Addons](https://github.com/adopted-ember-addons) has adopted [ember-sortable](https://github.com/adopted-ember-addons/ember-sortable), an addon which provides Sortable UI primitives for Ember. The addon was created by [Jamie White](https://github.com/jgwhite) over four years again and he has been maintaining it all this time! The time has come however for the addon to get some new love ❤ under the Adopted Ember Addons umbrella. 
```

Comment:
The project's language linter returns the following error in the [Travis CI run](https://travis-ci.org/ember-learn/ember-blog/builds/590426003):

<img width="1025" alt="Screenshot 2019-09-27 at 16 09 30" src="https://user-images.githubusercontent.com/8811742/65775911-5a8fad00-e141-11e9-91eb-8a09930488e5.png">

Since it's difficult to change the addon's maintainer's name to resolve the error, I'd think it'd be best to add the comment to let the linter ignore that error:

```suggestion
<!--alex ignore white-->
[Adopted Ember Addons](https://github.com/adopted-ember-addons) has adopted [ember-sortable](https://github.com/adopted-ember-addons/ember-sortable), an addon which provides Sortable UI primitives for Ember. The addon was created by [Jamie White](https://github.com/jgwhite) over four years again and he has been maintaining it all this time! The time has come however for the addon to get some new love ❤ under the Adopted Ember Addons umbrella. 
```


What do you think?